libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -90,6 +90,12 @@ const AP_Param::GroupInfo AP_RSSI::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("CHAN_HIGH", 6, AP_RSSI, rssi_channel_high_pwm_value,  2000),
 
+    // @Param: SCALE
+    // @DisplayName: Scale factor for rssi signal
+    // @Description: Scale factor for rssi signal
```

Comment:
We need to do better on the description here.  My understanding, for example, it that it is not easy for us to distinguish the scale for some receiver protocols so we need to provide this fudge-factor so people can correct it by hand.